.gitmodules

```diff
@@ -89,3 +89,12 @@
 	path = waterbox/dsda/core
 	url = https://github.com/TASEmulators/dsda-doom.git
 	branch = wbx
+[submodule "waterbox/ppsspp/ppsspp"]
+	path = ppsspp/ppsspp
+	url = https://github.com/TASEmulators/ppsspp.git
```

Comment:
Which branch?

Reply:
[ported2](https://github.com/TASEmulators/ppsspp/tree/ported2)

Reply:
It's better to always keep hawk specific changes rebased on upstream branch that we rely on. And each time we pull upstream and rebase anew we would increment our branch version like this: wbx-1, wbx-2, etc, so every rebase is a new branch, because it basically changes history, which we want to preserve on every update. This is better than merging upsream changes because it makes our changes harder to find over time.

And yeah please explicitly put the branch into this file, that way it's clear where the code is coming from. Only pointing to commit can result in detached head and that commit can belong to different branches, making future maintenance more annoying.

Reply:
added

Reply:
I can see the `wbx` branch in the repo now but not in `.gitmodules`...

Reply:
Uh, this is strange. This core isn't wbxed; it's ported. This every must be a residual from when I tried to wbx it

Reply:
Ah LOL indeed. Then the usual name for it is `bizhawk`.